testsuite/tests/tool-debugger/basic/debuggee.ml

```diff
@@ -2,13 +2,15 @@
 set foo = "bar"
 flags += " -g "
 ocamldebug_script = "${test_source_directory}/input_script"
-* debugger
-** shared-libraries
-*** setup-ocamlc.byte-build-env
-**** ocamlc.byte
-***** check-ocamlc.byte-output
-****** ocamldebug
-******* check-program-output
+* skip
+reason = "ocamldebug is broken (#34)"
```

Comment:
Is the debugger still broken?

Reply:
The debugger was broken, and we didn't attempt to fix it. It is likely to be broken. I quickly tested this and this test is failing:

```bash
$ make one TEST=tests/tool-debugger/basic/debuggee.ml
Test debugger should have depth atmost 1 but has depth 2
```